ui/src/components/facets/HistogramFacet.js

```diff
@@ -2,8 +2,6 @@ import React, { Component } from "react";
 import VegaLite from "react-vega-lite";
```

Comment:
Looks like we're only using react-vega-lite and vega-tooltip. Can you delete everything from package.json (react-vega, vega-embed, vega-lib, vega-lite)?

Reply:
Done.